packages/back-end/src/models/OrganizationModel.ts

```diff
@@ -303,25 +303,21 @@ export async function findOrganizationByStripeCustomerId(id: string) {
   return doc ? toInterface(doc) : null;
 }
 
-export async function getAllInviteEmailsInDb() {
+export async function getAllOrgMemberInfoInDb() {
```

Comment:
Might want to add an explicit return type here since it's hard to infer from the Mongo query projection.